From: Anuradha Weeraman <anuradha@debian.org>
Date: Tue, 24 Aug 2021 20:11:28 +0530
Subject: Fix FTBFS with autoconf 2.71

autogen.sh was incorrectly identifying pkg-config as
missing due to warnings of unused variables by autoconf.

Forwarded: not-needed
Comment: fixed differently upstream
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index a06ff30..9c98660 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -21,7 +21,7 @@ automake --add-missing --foreign
 # should be the case provided pkg-config is installed AND the above commands
 # have been run to prep the source tree with local set-up. 
 CHECK_PKG_CONFIG_M4='m4_ifdef([PKG_CHECK_MODULES], [errprint([ok])])'
-if [ "x$(autoconf <(echo "$CHECK_PKG_CONFIG_M4") 2>&1)" != "xok" ] ; then
+if [ "x$(autoconf --warnings=none <(echo "$CHECK_PKG_CONFIG_M4") 2>&1)" != "xok" ] ; then
   echo 'pkg-config appears to be missing (not available to autoconf tools)'
   echo 'please install the pkg-config package for your system.'
   exit 1
